pkgs/by-name/gi/git-credential-email/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  withOpencv ? false,
```

Comment:
Why is that nowhere enabled?

Reply:
OpenCV is an optional dependency for `git-protonmail` to automatically solve captchas. As the RPM package also only `Suggests` it, I have refrained from enabling it by default: https://github.com/AdityaGarg8/git-credential-email/blob/main/rpm/git-protonmail.spec#L35